.github/workflows/yaml-lint.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: GH Action YAML linter
+
+on:
+  pull_request:
+    paths:
+      - '.github/**.yml'
+      - '.github/**.yaml'
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    name: YAML Lint
+    runs-on: ubuntu-20.04
+    steps:
+      - name: LINT | Execute YAML linter
+        # yamllint disable-line rule:line-length
```

Comment:
Why do we need to diable rule? Below line has 64 characters, and max is 100 (https://github.com/Cardinal-Cryptography/github-actions/blob/main/yaml-lint/.yamllint#L45)